tests/test_parser.py

```diff
@@ -533,10 +537,28 @@ def test_elasticsearch_flag(self):
                 "pid": "number",
                 "string_array": ["string"],
                 "obj_array": ["string"],
-                "process": {"name": "string"}
+                "opcode": "number",
+                "process": {"name": "string"},
+                "unique_pid": "string"
+            },
+            "file": {
+                "opcode": "number",
+                "unique_pid": "string"
             }
         })
 
+        with elasticsearch_syntax:
+            subquery1 = '[process where opcode == 1] by unique_pid'
+            runs = [1, 2, 10, 30]
+            for run in runs:
+                subquery2_runs = '[file where opcode == 0] by unique_pid with runs={}'.format(run)
+                parse_query('sequence {} {}'.format(subquery1, subquery2_runs))
+
+            self.assertRaises(EqlSyntaxError, parse_query,
+                              'sequence [file where true] by field until [file where true] with runs=2')
+            self.assertRaises(EqlSemanticError, parse_query, 'sequence [process where opcode == 1] with runs=0')
+            self.assertRaises(EqlSyntaxError, parse_query, 'sequence [process where opcode == 1] with runs=-1')
```

Comment:
can you add a test to ensure `runs=1` totaling 1 total subquery fails

`sequence [process where opcode == 1] with runs=1`